### PR TITLE
Post launch tweeks

### PR DESCRIPTION
**Pull request by JosephMcArthur**
For https://github.com/oaworks/internal-planning/issues/109

### CODE REVIEW

File: src/policies/privacy.md [hunk 1]
```diff
@@ -18,19 +18,19 @@ Cookies are used for our account system. Users remain logged in but this can be
 
 ## Our Use of Your Data
 
-In general, the Open Access Button uses the data collected to further its Vision and Mission. We use user information to manage your membership and contact authors on your behalf. We may occationally use your email address to send you updates and alerts on Open Access and Open Access Advocacy. In addition to the metadata of the research papers, usernames, professions, and approximate locations will be publicly displayed in our data and website. To protect your privacy, the location data we collect is blurred by 11 kilometres.
+In general, the OA.Works uses the data collected to further its Vision and Mission. We use user information to manage your membership and contact authors on your behalf. We may occationally use your email address to send you updates and alerts on Open Access and Open Access Advocacy. In addition to the metadata of the research papers, usernames, professions, and approximate locations will be publicly displayed in our data and website. To protect your privacy, the location data we collect is blurred by 11 kilometres.
```
Comment (sssoz) [inline src/policies/privacy.md:21]: remove "the" from The OA.Works

Comment (sssoz) [reply src/policies/privacy.md:21]: American spelling is "11 kilometers"

File: src/policies/privacy.md [hunk 2]
```diff
@@ -18,19 +18,19 @@ Cookies are used for our account system. Users remain logged in but this can be
...
 
 ## App Downloads
 
 The Open Access Button can be downloaded from Mozilla Addons, Chrome Web Store. We encourage users to read the privacy policies of these stores. The Open Access Button can also be used directly from our website.
 
 ## Disclosure of Your Data
 
-We will provide relevant data for use by advocates to further the mission of the Open Access Button. This data will be released under a [Creative Commons 0 licence](https://creativecommons.org/publicdomain/zero/1.0/).
+We will provide relevant data for use by advocates to further the mission of the OA.Works. This data will be released under a [Creative Commons 0 licence](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (sssoz) [inline src/policies/privacy.md:33]: remove "the" from "the OA.Works"

File: src/newsletter/thanks.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: "layouts/default.liquid"
 theme: dark
-title: "We've got your sign up."
+title: "We've got your signup."
```
Comment (sssoz) [inline src/newsletter/thanks.md:4]: Remove fullstop

File: src/policies/privacy.md [hunk 4]
```diff
@@ -40,15 +40,16 @@ You may choose to correct, update, or delete the information you have submitted
...
 
 ## Changes to Our Policies
 
 This policy is current as of October 19th, 2016. Our Privacy Policy may change from time to time. However, any revised privacy policy will be consistent with our mission. If we make any substantive changes to our policies, we will place notice in our blog, post notice of changes on this page, and email users.
 
 ### Updates
 
-- October 24, 2016: Updated to reflect changes in the Open Access Button team, how we use data, and make structural improvements to the page.
+- October 24, 2016: Updated to reflect changes in the OA.Works team, how we use data, and make structural improvements to the page.
+- April 28th 2021: Updated our "Open Access Button" throughout to reflect our new name. We did not post notices as this wasn't deemed a substantial change.
```
Comment (sssoz) [inline src/policies/privacy.md:52]: Most recent should be at the top

Comment (sssoz) [reply src/policies/privacy.md:52]: > Updated our "Open Access Button"

"Updated the name _Open Access Button_ ...

File: src/policies/terms.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
 ---
 layout: "layouts/default.liquid"
 theme: dark
-title: Terms of Service
+title: Terms of service
 meta_description: "These terms of service cover your use and access to the services."
 ---
 
-These terms of service cover your use and access to the services ("the service"), software ("the software") and websites ("the website") provided by the Open Access Button. Our Privacy Policy explains how we collect and use your information. By using our service, you are agreeing to be bound by these Terms and to accept our [Privacy Policy](/policies/privacy).
+These terms of service cover your use and access to the services ("the service"), software ("the software") and websites ("the website") provided by the OA.Works. Our Privacy Policy explains how we collect and use your information. By using our service, you are agreeing to be bound by these Terms and to accept our [Privacy Policy](/policies/privacy).
```
Comment (sssoz) [inline src/policies/terms.md:8]: remove "the" from "the OA.Works"

File: src/policies/terms.md [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
 
 ## Registration
 
 You may browse the website and view content without registering, but as a condition to using certain aspects of the service, you are required to register with a valid email address. You are solely responsible and liable for activity that occurs on your account.
 
 ## Acceptable Use Policy
 
-We trust you to use our services responsibly. You agree not to misuse the Open Access Button services ("Services"). For example, you must not, and must not attempt to do the following:
+We trust you to use our services responsibly. You agree not to misuse the OA.Works services ("Services"). For example, you must not, and must not attempt to do the following:
```
Comment (sssoz) [inline src/policies/terms.md:16]: remove "the" from "the OA.Works"

File: src/policies/terms.md [hunk 7]
```diff
@@ -22,14 +22,14 @@ We trust you to use our services responsibly. You agree not to misuse the Open A
...
+OA.Works also provides links to or interacts with a wide variety of third-party websites, including interactive links to sites like social media via application programming interfaces (APIs). When you access third party websites, you do so at your own risk. These other websites are not under the control of the OA.Works, and you acknowledge that OA.Works is not responsible or liable for the content, functions, accuracy, legality, appropriateness, security or any other aspect of such websites or resources. The inclusion of any such link does not imply endorsement by OA.Works.
```
Comment (sssoz) [inline src/policies/terms.md:25]: remove "the" from "the OA.Works"

Comment (sssoz) [reply src/policies/terms.md:25]: Hyphenate _third-party websites_

File: src/policies/terms.md [hunk 8]
```diff
@@ -22,14 +22,14 @@ We trust you to use our services responsibly. You agree not to misuse the Open A
...
+All data and information provided on this site is for informational purposes only. While New Venture Fund and its Scholarly Publishing and Academic Research Coalition ("SPARC") project have used their best efforts to ensure that the information provided on this site is accurate and complete, they make no legal representations as to the accuracy, completeness, suitability or validity of any data or information on this site and will not be liable for any errors or omissions in this data and information or for any losses, injuries or damages arising from its display or use. All information is provided on an as-is basis. The OA.Works is an initiative of the Scholarly Publishing and Academic Research
```
Comment (sssoz) [inline src/policies/terms.md:27]: remove "the" from "the OA.Works"

File: src/policies/terms.md [hunk 9]
```diff
@@ -22,14 +22,14 @@ We trust you to use our services responsibly. You agree not to misuse the Open A
...
 Coalition ("SPARC"), a project of the New Venture Fund.
 
 ## Copyright/Intellectual Property
 
-We respect the intellectual property of others and ask that you do too. Through the use of the service, if you directly or indirectly share research with others that you do not have the appropriate rights to share, the Open Access Button will not be held liable for the consequences of those actions.
+We respect the intellectual property of others and ask that you do too. Through the use of the service, if you directly or indirectly share research with others that you do not have the appropriate rights to share, the OA.Works will not be held liable for the consequences of those actions.
```
Comment (sssoz) [inline src/policies/terms.md:32]: remove "the" from "the OA.Works"

File: src/policies/terms.md [hunk 10]
```diff
@@ -41,6 +41,7 @@ This policy is current as of February 27th, 2020. Our Privacy Policy may change
 
 - October 24th, 2016: Updated text to remove references to passwords. We no longer have them in favour of a more secure password less system. This page structure was also updated to mimic our privacy policy so there is now a clear place to ask questions and understand our update policy. We also cleaned up some formatting and removed the "welcome message" at the start of the policy. This was not posted to our blog as it wasn't deemed a substantial change.
 - February 27th 2020: Updated terms with an additional standard disclaimer to support Shareyourpaper.org. We did not post notices as this wasn't deemed a substantial change, and users of that service will be asked to accept these separately.
+- April 28th 2021: Updated our "Open Access Button" throughout to reflect our new name. We did not post notices as this wasn't deemed a substantial change.
```
Comment (sssoz) [inline src/policies/terms.md:44]: > Updated our "Open Access Button"

"Updated the name _Open Access Button_ ...

**Review by sssoz — CHANGES_REQUESTED** [review]
Reviewed punctuation and grammar.

**Comment by JosephMcArthur** [comment]
Thanks! Changes made!

**Pull request merged**